test/fast-list-test.js

```diff
@@ -136,22 +135,22 @@ suite('FastList >', function() {
             to: 2
           });
         });
-      });
 
-      suite('then grows back', function() {
-        setup(function() {
-          var bigData = createDummyData(1000);
-          source.data = bigData;
-          fastList.reloadData();
-          scheduler.mutation.yield();
-        });
+        suite('then grows back', function() {
```

Comment:
my style of nested unit tests usually serves me well, but I missed a nesting level here before :)